docs/roadmap.rst

```diff
@@ -65,15 +51,13 @@ Goal
 ++++
 
 Adhocracy should be optimized to allow processes to be administered by
-initiators to take the workload off of our project managers and
-developers. It also should optimize platform features that allow users
+initiators. It also should optimize platform features that allow users
 and organizations to feel more at home.
 
 KPIs
 ++++
 
 -   we can offer an initial set up of a new process and organization
-    with little effort
```

Comment:
If you leave this out it is not clear how this is different from the other milestones.